docs/user-guide/petset.md

```diff
@@ -0,0 +1,370 @@
+---
+---
+
+* TOC
+{:toc}
+
+__Terminology__
+
+Throughout this doc you will see a few terms that are sometimes used interchangeably elsewhere, that might cause confusion. This section attempts to clarify them.
+
+* Node: A single virtual or physical machine in a Kubernetes cluster.
+* Cluster: A group of nodes in a single failure domain, unless mentioned otherwise.
+* Persistent Volume Claim: A request for storage, typically a [persistent volume](/docs/user-guide/persistent-volumes/walkthrough.md).
+* Host name: The hostname attached to the UTS namespace of the pod, i.e the output of `hostname` in the pod.
+* DNS/Domain name: A *cluster local* domain name resolvable using standard methods (eg: [gethostbyname](http://linux.die.net/man/3/gethostbyname)).
+* Ordinality: the proprety of being "ordinal", or occupying a position in a sequence.
+* Pet: a single member of a Pet Set; more generally, a stateful application.
+* Peer: a process running a server, capable of communicating with other such processes.
+
+## What is a Pet Set?
+
+In Kubernetes, most pod management abstractions group them into disposable units of work that compose a micro service. Replication controllers for example, are designed with a weak guarantee - that there should be N replicas of a particular pod template. The pods are treated as stateless units, if one of them is unhealthy or superseded by a better version, the system just disposes it.
```

Comment:
First sentence is hard to understand.  `existing abstractions group pods as identical [fungible?] units of work` is better, but disposable is weird.